css/index.css

```diff
@@ -170,43 +170,43 @@ a {
 }
 
 .middle-content .boxes .box:first-child {
-    background-color: teal;
+    background-color: #008080;
 }
 
 .middle-content .boxes .box:nth-child(4n+2) {
-    background-color: gold;
+    background-color: #FFD700;
```

Comment:
I love how clever your solution is for this objective. Let me provide an alternate way of approaching this problem, simply for consideration. 

Our current html code looks like this:
```<div class='box'>box1</div>```

Another way of approaching this problem is to simply add an additional class:
<div class='box box1'>box1</div>

This will allow your css to be more descriptive in case you need to find a particular box

CSS:
``` .box1{
      background: teal; 
    }
```

Your solution isn't incorrect, this is simply another way of looking at the problem.